src/workerd/server/tests/python/sdk/worker.py

```diff
@@ -0,0 +1,84 @@
+from http import HTTPMethod
+
+from cloudflare.workers import Response, fetch
+
+
+async def on_fetch(request):
+    if request.url.endswith("/modify"):
+        resp = await fetch("https://example.com/sub")
+        return Response(
+            resp.body,
+            status=201,
+            headers={"Custom-Header-That-Should-Passthrough": "modified"},
+        )
+    elif request.url.endswith("/modify_tuple"):
+        resp = await fetch("https://example.com/sub")
+        return Response(
+            resp.body,
+            headers=[
+                ("Custom-Header-That-Should-Passthrough", "modified"),
+                ("Custom-Header-That-Should-Passthrough", "another"),
+            ],
```

Comment:
What does the code look like if we want to add extra headers? Is that convenient?

Reply:
How do you mean? You can just add extra tuples into the list or use a dictionary (like above).

This is testing that duplicate headers can be specified and most shouldn't use this form when specifying headers.